### PR TITLE
changing repetition of code to foreach

### PR DESCRIPTION
**Pull request by yurastico**

### CODE REVIEW

**Review by bdach — CHANGES_REQUESTED** [review]
Please trim unnecessary whitespace as per CI suggestions. You can [run code analysis locally](https://github.com/ppy/osu#code-analysis) to ensure style correctness before pushing again.

File: osu.Game/OsuGameBase.cs [hunk 1]
```diff
@@ -134,29 +155,33 @@ private void load()
...
+
+
+
+
+
+
+
+
+
+
```
Comment (TPGPL) [inline osu.Game/OsuGameBase.cs]: you should probably delete all this whitespace

**Review by TPGPL — CHANGES_REQUESTED** [review]

**Review by frenzibyte — CHANGES_REQUESTED** [review]
If this is to be corrected for, It should probably be looking up for any font inside the `Fonts` folder rather than having a hardcoded list containing the fonts.

File: osu.Game/OsuGameBase.cs [hunk 2]
```diff
@@ -120,6 +120,27 @@ public OsuGameBase()
 
         protected override UserInputManager CreateUserInputManager() => new OsuUserInputManager();
 
+        private readonly IList<string> fontPaths = new List<string>()
```
Comment (swoolcock) [inline osu.Game/OsuGameBase.cs]: Use `IEnumerable<string>` and an array.

**Comment by peppy** [comment]
Closing as low quality / abandoned. Please re-open if you wish to fix this.

**Pull request closed without merging**